docker-compose.yml

```diff
@@ -1,100 +1,69 @@
 version: "2"
 
 services:
-  resolvable:
-    image: mgood/resolvable
-    volumes:
-      - "/var/run/docker.sock:/tmp/docker.sock"
-      - "/etc/resolv.conf:/tmp/resolv.conf"
-
```

Comment:
This is very handy in the development mode, since you don't have to search for the container IP address if you want to connect to the postgres database or point your browser to some endpoint.